sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuCreateDataSourceTableAsSelectCommand.scala

```diff
@@ -36,7 +36,8 @@ case class GpuCreateDataSourceTableAsSelectCommand(
     query: LogicalPlan,
     outputColumnNames: Seq[String],
     origProvider: Class[_],
-    gpuFileFormat: ColumnarFileFormat)
+    gpuFileFormat: ColumnarFileFormat,
+    useStableSort: Boolean)
```

Comment:
My only nit is that we pass useStableSort around this code a lot, but in the final part when we do the sort we get it from a different location.

https://github.com/NVIDIA/spark-rapids/blob/4a313e7046d0aa34f284483a5ed643d9cb8b79f5/sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuFileFormatWriter.scala#L207-L211

Could we please make it consistent? Either we pass it all the way down all the time, or we go off of the plan.conf all the time.